RELEASE.md

```diff
@@ -1,36 +1,36 @@
 
-OpenEBS follows a reasoably agressive release cadence.<BR>
----
-- While in the CNCF ``` Sandbox level ``` We try to relase features, fixes and updates as frequently as possibe.
-- As we progress up through the CNCF levels towards ``` Graduation ```, we expect the project to mature and the release cadence to change in alignment with CNCF guidelines.
 ---
+### **Planning & Feature Freeze**
+- The release cycle begins with a planning phase, where maintainers and contributors define the roadmap and key features for the upcoming release.
+- A feature freeze is enforced to ensure only bug fixes and critical changes are allowed beyond this point.
 
-### Defining a release
-Our current release mechanisim is managed under the following methdology; where the scope of each release is determined, influenced and stuctured by the following process. (in priority order):
+### **Development & Code Merging**
+- Developers work on new features, improvements, and bug fixes.
+- Changes are reviewed via GitHub pull requests (PRs), passing automated tests before merging across all subprojects.
 
-1. Pending items listed in the [roadmap](./ROADMAP.md)
-2. Approved ``` community contributor ``` PR's
-3. Approved ```core Dev/Eng/QA ``` team PR's
-4. Approved ```Primary Project Sponsor ``` PR's
-5. Maintainer approved OEP's (openEBS Enhancment Proposal)
-6. [GitHUb ISSUES filed by the community](https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Aproject%2Fcommunity+org%3Aopenebs&type=Issues)
-7. [GitHUb ISSUES filed by the Eng/Dev/QA team](https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Aproject%2Fcommunity+org%3Aopenebs&type=Issues)
-8. [GitHUb ISSUES filed by the Maintainers](https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Aproject%2Fcommunity+org%3Aopenebs&type=Issues)
-9. Any special, emergency items (ISSUES/PR's) requested by ``` Special Maintainers ```:
-   - (**Director Maintainers, CNCF Liasons, Proj/Prog Manager Maintainers, Sub-Project Maintainers**)
+### **Pre-Release**
+- OpenEBS uses continuous integration (CI) to run unit tests, integration tests, and end-to-end (E2E) tests.
+- Performance and compatibility testing is conducted across multiple Kubernetes versions and storage backends.
+- Since OpenEBS has multiple Sub-Projects each of the individual subprojects follow their own pre-release process. Refer to the below ones for more detail:
+   * ZFS LocalPV - https://github.com/openebs/zfs-localpv/blob/develop/RELEASE.md
+   * LVM LocalPV - https://github.com/openebs/lvm-localpv/blob/develop/RELEASE.md
+   * Hostpath LocalPV - https://github.com/openebs/dynamic-localpv-provisioner/blob/develop/RELEASE.md
+   * Mayastor - https://github.com/openebs/mayastor/blob/develop/RELEASE.md
+- Once all the Sub-Projects pre-release tags are available, on OpenEBS umbrella repository the respective release branch is created. Ex `release/x.y`.
```

Comment:
hmm this isn't accurate for the umbrella today or is it?

Reply:
No it isn't this file needs to be refactored once we decide the actual process. I just tried to put a rough idea.